docs/akr.md

```diff
@@ -129,14 +129,14 @@ mvn spring-boot:run -Dtomcat.util.http.parser.HttpParser.requestTargetAllow=|{}
 
 I18next is used as an internationalization framework. Localizations are stored in JSON files and committed to git.
```

Comment:
tännäkin voisi laittaa haluttaessa --> `stored in JSON files in public/i18n directory` ...